cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -43,6 +43,7 @@ on:
   push:
     tags:
       - '*-?v[0-9]+*'
+      - '*/?v[0-9]+*'
```

Comment:
hmm probably preferable for to just be folded into the same pattern..?

Reply:
also do we really need/want to *require* the `v` prefix at this point? (does anyone else you know like using this scheme, do they like keeping the v?)

Reply:
I initially tried keeping it in the same pattern, but it seems workflow patterns are not quite as powerful as normal RegEx, so it failed when I had `*[\-/]?v[0-9]+*'` unfortunately :(

As for the `v` prefix, it might be an idea to not require it. Personally I always like using it to make it easier to filter tags, but in some projects at work we do not use it, so there is a precedent for not having it.

Making the `v`-prefix optional would require changes to `tasks.rs` again (and the documentation). But with probably a smarter solution than just adding another two branches to the `if`-statement, otherwise it will get unwieldy.

What are your thoughts?

Reply:
Hey sorry for the delay, got sick.

I think you're right that doing one pattern won't work. 

And I agree the current tag parsing code is lackluster, I'll push a refactor to the tag code to this PR (my fault it's this bad 😅).

Reply:
Ah no worries, hope you're feeling better now!

I'll wait for your changes then :) And then see about making the `v`-prefix optional for both the slash and non-slash tags? (Maybe there's a usecase for people who tag like `my-app-1.2.3`.)